src/staticanalysis/bot/static_analysis_bot/coverity/coverity.py

```diff
@@ -256,8 +256,16 @@ def is_publishable(self):
         as we do with the rest of our Analyzers, for IN_PATCH or BEFORE_AFTER methods,
         since Coverity performs most of the checks on the servers and provides us a
         snapshot with the checks that can be filtered only by the is_local function.
+        Coverity also has the ability to forward clang errors but we don't want to forward
+        these errors to the review.
         '''
-        return self.is_local()
+        return not self.is_clang_error() and self.is_local()
+
+    def is_clang_error(self):
+        '''
+        Determine if the current issue is a translation unit error forwarded by Clang
+        '''
+        return 'RW.CLANG' in self.kind
```

Comment:
You can use `self.kind.starswith('RW.CLANG')` if you only want to match the beginning